app/lib/frontend/handlers/routes.dart

```diff
@@ -360,20 +369,8 @@ class PubSiteService {
   Future<Response> taskLog(
       Request request, String package, String version) async {
     checkPackageVersionParams(package, version);
-
-    InvalidInputException.checkPackageName(package);
-    version = InvalidInputException.checkSemanticVersion(version);
-
-    if (!await packageBackend.isPackageVisible(package) ||
-        (await packageBackend.lookupPackageVersion(package, version)) == null) {
-      return Response.notFound('no such package');
-    }
-
-    final log = await taskBackend.taskLog(package, version);
-    return Response.ok(
-      log ?? 'no log',
-      headers: {'content-type': 'plain/text'},
-    );
```

Comment:
I propose that we just remove this end-point, or at that we at-least leave a TODO for it to be deleted in the future.